### DISCUSSION TITLE
用Flask架網站輸入與輸出

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#258] [Q&A] [answered]
最近在學習如何用flask架設網站，但遇到了一個難題就是，不知該如何讓使用者在我們的網站上輸入的內容，直接回傳變成後端程式執行時的輸入值，然後程式再跑出結果給網站的使用者。
目前我們的進度停在網站能讓使用者輸入內容，但輸入的內容不知道會跑去哪裡，也不會回傳並提供給後端程式執行，以下是我們學習的參考網站。
[https://pythonbasics.org/flask-tutorial-routes/](https://pythonbasics.org/flask-tutorial-routes/)

**Comment by chun1203** [comment]
我們最近通識課也要架網站，對於程式基礎薄弱的我真的是一大困難🥲

**Reply by chia-kai-chang** [reply]
哪一堂通識課要架設網站? 😂

**Reply by charlottechuang** [reply]
機器學習概論

**Reply by zixunn** [reply]
> 哪一堂通識課要架設網站? 😂

那老師你會嗎Q^Q

**Reply by chia-kai-chang** [reply]
你們還沒出生，大叔就在寫網站了
是不漂亮啦，馬馬虎虎~
https://labview360.org/
https://i.cckai.org/labview360-community/li-dai-labview360-zhu-ji

**Comment by charlottechuang** [comment, marked as answer]
route是指將網頁導到某個介面 ~所以除了定義一個route是可以讓用戶輸入之外，還需要def一個新的route取得用戶輸入的資料並導到你們所做的預測模型讓他可以return(回傳)預測數值。
